src/util/prUtils.ts

```diff
@@ -1,9 +1,14 @@
 import * as ghCore from "@actions/core";
 import * as github from "@actions/github";
+import { Octokit } from "@octokit/core";
 
 import Crda, { GIT_EXECUTABLE } from "../crda";
+import { Inputs } from "../generated/inputs-outputs";
 import { CrdaLabels } from "./constants";
 import * as labels from "./labels";
+import { getBetterHttpError } from "./utils";
+
+let ghToken: string | undefined;
```

Comment:
move this down to the function that uses/sets it so it's more clear what it does